src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
+          project_id: s.project.id,
+          timeToFinish: (12 - moment.utc().diff(s.assigned_at, 'hours')),
```

Comment:
I thought this was being updated everytime a `checkAssigned` was run, which is run every 5 minutes because it is called from inside `submissionRequest`. Then the `assigned` array is updated. 

Did I get the logi wrong? :confused: